.github/workflows/build.yml

```diff
@@ -254,7 +254,7 @@ jobs:
           name: plugins
           path: zips
           # Only need to retain for a day since the beta builder slurps it up to distribute.
-          retention-days: 1
+          retention-days: ${{ github.ref == 'refs/heads/trunk' && 7 || 1 }}
```

Comment:
Either update the comment here too, or just leave this one at 1 day for everything.

This one really only exists so betadownload.jetpack.me doesn't have to do a bunch of processing of the jetpack-build artifact to produce the zips, instead we do all the processing in Actions and betadownload.jetpack.me only has to save the zips to the right places and update its metadata json files.

Reply:
Yeah, I was going for consistency across the build, but I guess there's probably no need for it.